src/components/views/sul-branch-locations/filtering-table/branch-locations-filtering-table.tsx

```diff
@@ -126,62 +128,74 @@ const TableRow = ({id, imageUrl, path, title, phone, email, mapUrl, address, hou
       className="block sm:flex-col sm:flex-wrap sm:text-center md:grid md:grid-cols-2 md:grid-rows-[repeat(4,minmax(0,auto))] md:justify-items-start md:gap-x-20 md:text-left md:align-top lg:!table-row lg:max-h-none"
     >
       <Td className="table-image m-auto flex min-h-fit w-auto place-content-center justify-center sm:border-b sm:border-black-40 md:row-span-4 lg:mr-25 lg:table-cell lg:w-[125px] lg:pr-16 lg:align-middle">
-        <div className="relative block aspect-[3/2] w-auto max-w-[338px] overflow-hidden md:w-[290px] lg:max-w-[125px]">
-          {imageUrl && (
+        {imageUrl && (
+          <div className="relative my-16 block aspect-[3/2] w-auto max-w-[338px] overflow-hidden md:w-[360px] lg:max-w-[125px]">
             <Link
```

Comment:
FWIW, you could actually eliminate the `<div>` tag entirely and apply styles to the `<Link>` element.

Reply:
ok. got it.